utils/deterministic_graph_data.py

```diff
@@ -40,22 +45,33 @@ def random_graph_data(
 
     ###############################################################################################
 
+    number_atoms = 2 * number_unit_cell_x * number_unit_cell_y * number_unit_cell_z
+    positions = torch.zeros(number_atoms, 3)
+    lattice_parameter = 0.5
+
+    # We assume that the unit cell is Body Center Cubic (BCC)
+    count_pos = 0
+    for x in range(0, number_unit_cell_x):
+        for y in range(0, number_unit_cell_y):
+            for z in range(0, number_unit_cell_z):
+                positions[count_pos][0] = x + 0.0
+                positions[count_pos][1] = y + 0.0
+                positions[count_pos][2] = z + 0.0
+                positions[count_pos + 1][0] = x + lattice_parameter
```

Comment:
Should be + 1/2 the lattice parameter for BCC

Reply:
I already use 0.5. I set `lattice_parameter = 0.5` a few lines above. 
The unit cell has 0.5 as standard lattice size. Then this is pre-multiplied by the lattice parameter that attains the minimum energy configuration for each material. But in this case I would keep the standard lattice size of the unit cell, which is 0.5. I already use 0.5 for the unit test.

Reply:
Sorry, I misread. I would just use `+0.5` directly to avoid confusion - eventually we can rewrite this to use any lattice basis